Makefile

```diff
@@ -341,7 +341,7 @@ beautify: virtualenv/bin/fixjsstyle
 spritegen: $(SPRITE_GENERATED_FILES)
 
 $(COMPILED_JSX_FILES): $(JSX_FILES)
-	jsx --no-cache-dir -x jsx . .
+	# jsx --no-cache-dir -x jsx . .
```

Comment:
I felt that just removing the line of code will leave some odd hanging code remaining and I didn't want the branch to creep to much to fully remove the jsx stuff so I elected to comment it out as a interim step.